example/AcceleratedForwardFFT.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ITKVkFFT Python Demonstration\n",
+    "\n",
+    "This example is derived from the \"Compute Forward FFT\" ITK Sphinx example at https://itk.org/ITKExamples/src/Filtering/FFT/ComputeForwardFFT/Documentation.html to demonstrate FFT performance speedup with `itk-vkfft`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "!{sys.executable} -m pip install itk itk-vkfft matplotlib"
```

Comment:
I don't think you need to list `itk`. It should be installed as a dependency of `itk-vkfft`.

Reply:
Removed